docs/index.rst

```diff
@@ -315,3 +315,48 @@ Configuration Options
   * Type: long
   * Default: 60000
   * Importance: low
+
+
+Schema Evolution
+----------------
+
+The JDBC connector supports schema evolution. When there is a change in a database table schema,
```

Comment:
Oh, one last point I realized -- do we want to mention that this is specific to using `AvroConverter`? If we're not using `AvroConverter`, then obviously a lot of this doesn't apply... The first set of statements up to the new Kafka Connect schema make sense, but everything after assumes AvroConverter.